qemu/tests/rh_kernel_update.py

```diff
@@ -92,7 +92,7 @@ def get_kernel_info():
             pkg_arch = pkg_params["pkg_arch"]
             # package pattern: n-v-r.a
             pkg_pattern = "%s-%s.%s." % (pkg_name, ver_rev, pkg_arch)
-            pkg_pattern = re.compile(".*/%s.*" % re.escape(pkg_pattern))
+            pkg_pattern = re.compile(r".*/%s\S*" % re.escape(pkg_pattern))
```

Comment:
How about use fixed string `rpm` ?

Reply:
Currently, the output format is split by tab.
```
/mnt/redhat/brewroot/vol/rhel-7/packages/kernel/3.10.0/1127.el7/ppc64le/kernel-3.10.0-1127.el7.ppc64le.rpm	Signatures: fd431d51
```
If you think to add the fixed `rpm` will better, I will update it.

Reply:
Yes, IMO, no matter what the extra string is, we only need get the `rpm` to generate `pkg_path`.